driver/razerkbd_driver.c

```diff
@@ -1734,22 +1841,33 @@ static int razer_raw_event(struct hid_device *hdev, struct hid_report *report, u
                 found_fn = 0x01;
                 break;
             case 0x20: // M1
-                cur_value = 0x68; // F13
+                cur_value = USB_HID_KEY_F13; // F13
                 break;
             case 0x21: // M2
-                cur_value = 0x69; // F14
+                cur_value = USB_HID_KEY_F14; // F14
                 break;
             case 0x22: // M3
-                cur_value = 0x6A; // F15
+                cur_value = USB_HID_KEY_F15; // F15
                 break;
             case 0x23: // M4
-                cur_value = 0x6B; // F16
+                cur_value = USB_HID_KEY_F16; // F16
                 break;
             case 0x24: // M5
-                cur_value = 0x6C; // F17
+                cur_value = USB_HID_KEY_F17; // F17
+                break;
+            case 0x52: // Mute
+                cur_value = USB_HID_KEY_MEDIA_MUTE;
+                break;
+            case 0x53: // Next (song)
+                cur_value = USB_HID_KEY_MEDIA_NEXTSONG;
+                break;
+            case 0x55: // Play/Pause
+                cur_value = USB_HID_KEY_MEDIA_PLAYPAUSE;
+                break;
+            case 0x54: // Prev (song)
+                cur_value = USB_HID_KEY_MEDIA_PREVIOUSSONG;
                 break;
```

Comment:
I hope these new cases don't break existing keyboards...

Reply:
The code may effect only for *keyboards*, because original code with special scan codes 0x20..0x24 works for all keyboards (even boards without M1-M5 buttons) without any issues - I , suppose that handling special scan codes 0x52..0x54 for multimedia buttons should work for keyboards without multimedia buttons too